libs/shared/adapters/src/lib/cms-cache-adapter.ts

```diff
@@ -887,13 +850,17 @@ export class CMSCacheAdapter {
       queryBuild = queryBuild.orderBy(sort.field, sort.order)
     }
 
-    return queryBuild.page(
-      query.page - 1 || 0,
-      query.limit != -1 ? query.limit : Number.MAX_SAFE_INTEGER
-    )
+    const page = query.page - 1 || 0
+    const pageSize =
+      query.limit != -1 ? query.limit || 10 : Number.MAX_SAFE_INTEGER
+
+    return queryBuild.page(page, pageSize)
```

Comment:
Not sure if this is the correct/expected behavior, but was seeing warning logs because the old version passed `undefined` as the `pageSize`.